src/mcp-server/CLAUDE_INTEGRATION.md

```diff
@@ -0,0 +1,155 @@
+# Claude Desktop Integration Guide
+
+## 🚀 Quick Start
+
+### 1. Prerequisites Setup
+
+**Start Prometheus Port Forwarding:**
+```bash
+# Run this first to enable Prometheus access
+cd src/mcp-server/scripts
+./setup-prometheus.sh
+```
+
+### 2. Claude Desktop Configuration
+
+Copy the configuration from `src/mcp-server/integrations/claude-desktop-config.json`:
+
+```json
+{
+  "mcpServers": {
+    "ai-observability": {
+      "command": "/Users/ahameed/projects/ai/kickstart/openshift-ai-observability-summarizer/.venv/bin/obs-mcp-server",
+      "args": ["--local"],
+      "env": {
+        "PROMETHEUS_URL": "http://localhost:9090"
+      },
+      "autoApprove": [
+        "list_models",
+        "get_namespaces",
+        "get_metrics",
+        "get_alerts"
+      ],
+      "alwaysAllow": [
+        "list_models",
+        "get_namespaces",
+        "get_metrics",
+        "get_alerts"
+      ],
+      "disabled": false
+    }
+  }
+}
+```
+
+### 3. Claude Desktop Configuration Location
+
+**macOS:** `~/Library/Application Support/Claude/claude_desktop_config.json`
+
+### 4. Test the Integration
+
+1. **Start Prometheus forwarding** (in terminal):
+   ```bash
+   cd /Users/ahameed/projects/ai/kickstart/openshift-ai-observability-summarizer/src/mcp-server/scripts
```

Comment:
Same comment regarding not using user specific path and making it a template, e.g. `<OBS_BASE_DIR>/src/mcp-server/scripts`

Reply:
done thanks